content/en/docs/ops/deployment/security-model/index.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Security Model
+description: Describes Istio's security model.
+weight: 10
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+This document aims to describe the security posture of Istio's various components, and how possible attacks can impact the system.
+
+## Components
+
+Istio comes with a variety of optional components that will be covered here.
+For a high level overview, see [Istio Architecture](/docs/ops/deployment/architecture/).
+Note that Istio deployments are highly flexible; below, we will primarily assume the worst case scenarios.
+
+### Istiod
+
+Istiod serves as the core control plane component of Istio, often serving the role of the [XDS serving component](/docs/concepts/traffic-management/) as well 
+as the mesh [mTLS Certificate Authority](/docs/concepts/security/).
+
+Istiod is considered a highly privileged component, similar to that of the Kubernetes API server itself.
+* It has high Kubernetes RBAC privileges, typically including `Secret` read access and webhook write access.
+* When acting as the CA, it can provision arbitrary certificates.
+* When acting as the XDS control plane, it can program proxies to perform arbitrary behavior.
+
+As such, the security of the cluster is tightly coupled to the security Istiod.
+Following [Kubernetes security best practices](https://kubernetes.io/docs/concepts/security/) around Istiod access is paramount.
+
+### Istio CNI plugin
+
+Istio can optionally be deployed with the [Istio CNI Plugin DaemonSet](docs/setup/additional-setup/cni/).
+This DaemonSet is responsible for setting up networking rules in Istio to ensure traffic is transparently redirected as needed.
+This is an alternative to the `istio-init` container discussed [below](#sidecar-proxies).
+
+Because the CNI DaemonSet modifies networking rules on the node, it requires an elevated `securityContext`.
+However, unlike [Istiod](#istiod), this is a **node-local** privilege.
+The implications of this are discussed [below](#node-compromise).
+
+### Sidecar Proxies
+
+Istio may [optionally](docs/overview/dataplane-modes/) deploy a sidecar proxy next to an application.
+
+The sidecar proxy needs the network to be programmed to direct all traffic through the proxy.
+This can be done with the [Istio CNI plugin](#istio-cni-plugin) or by deploying an `initContainer` (`istio-init`) on the pod (this is done automatically if the CNI plugin is not deployed).
+The `istio-init` container requires `NET_ADMIN` and `NET_RAW` capabilities.
+However, these capabilities are only present during the initialization - the primary sidecar container is completely unprivileged.
+
+Additionally, the sidecar proxy does not have any associated Kubernetes RBAC privileges at all.
```

Comment:
Unclear exactly what this is. Do you mean if the app container has certain RBAC priviledges and these are not conveyed to the sidecar proxy?  Or you mean the sidecar proxy doesn't require `NET_ADMIN` and `NET_RAW` capabilities.

Reply:
> Or maybe you mean sidecar proxy doesn't need any special kubernetes RBAC priviledge.

this one. Basically ssaying it has linux caps, but no k8s rbac